src/Components/ADT3DViewer/ADT3DViewer.stories.tsx

```diff
@@ -41,6 +45,10 @@ export const Engine = (_args, { globals: { theme, locale } }) => {
         </div>
     );
 };
+Engine.play = async ({ _canvasElement }) => {
+    //👇 This sets a timeout of 2s
+    await new Promise((resolve) => setTimeout(resolve, 2000));
```

Comment:
we might need to extend this timeout depending on the model file size, right, to wait it to load before taking the snapshot? if so, we can add a comment for it to remember if we change the mock file.

Reply:
I just set arbitrary time, but yeah, we might need to change this based on file size. I'll add a comment